### ISSUE TITLE
Containerized release and use of LANL and Microsoft. 

### ISSUE THREAD

**Issue by molikd** [ericleasemorgan/reader#112]
Our current plan in order to take advantage of all resources given to us is to auto-build #34 three containers (I think, need confirmation):

1. Web Server/Job Submission Container (reader-ws) 
2. A Job Running Container (reader-j)
3. Solr Index container (reader-s)

This means that for a given set of resources the web server will submit a job, which will spin up a container of the job container type, both will interact with Solr. 

Solr meanwhile will have to be configured to be cross datacenter. #101 

We will build two distant readers to start West and East, later we could add an India cluster. reader-ws will be deployed to Jetstream in East and Microsoft in West, reader-j will run on Slurm, Jetstream in East and LANL in West, Solr will run on Jetstream in East and Microsoft in West. 

This means that for me, what I should do is use the future West implementation as the test bed for containerization. Maybe starting with a simple https://azure.microsoft.com/en-us/resources/templates/slurm/ Slurm implementation in the West.

**Comment by ericleasemorgan** [comment]
Dave, on your mark. Get set. Go.

Do you need anything from me?

**Comment by molikd** [comment]
Yes, are the software dependencies up-to-date on the README.md?

**Comment by ericleasemorgan** [comment]
Yes, they are up-to-date, but there is no need for Arivata. Okay?
